exams/views.py

```diff
@@ -10,3 +18,27 @@ class PearsonCallbackRedirectView(RedirectView):
     """
     def get_redirect_url(self, status):  # pylint: disable=arguments-differ
         return "/dashboard?exam={status}".format(status=status)
+
+
+class PearsonSSO(APIView):
+    """
+    Views for the Pearson SSO API
+    """
+    authentication_classes = (
+        authentication.SessionAuthentication,
+        authentication.TokenAuthentication,
+    )
+    permission_classes = (permissions.IsAuthenticated, )
+
+    def post(self, request, *args, **kargs):  # pylint: disable=unused-argument, no-self-use
+        """
+        handle the post request
+        """
+        # import ipdb
+        # ipdb.set_trace()
+        student_id = request.data.get('student_id')
```

Comment:
Ah ok I just misunderstood and though I would generate those values on the frontend.